UKHomeOffice/cop-ui
# COP-4723 Refactor secure local storage into custom class

### AC
Local storage capability should be the same as before with a different implementation (task filters persist)

### Updates
- Rename SecureLocalStorage to SecureLocalStorageManager
- Place secure-ls logic into custom class (SecureLocalStorageManager)
- Added jest-localstorage-mock for testing local storage
- Add tests for SecureLocalStorageManager

### To test
- Pull and run cop locally
- Login
- Go to group tasks
- Change task filters
- *They should persist on refresh*
- Go to your own tasks
- Change task filters
- *They should persist on refresh*
